packages/at_client/test/notification_service_test.dart

```diff
@@ -521,55 +478,192 @@ void main() {
           .thenAnswer((_) => mockAtClientImpl);
     });
 
-    test('getLastNotificationTime() returns null if checkOfflineNotifications is set to false',
-            () async {
-          var notificationServiceImpl = await NotificationServiceImpl.create(
-              mockAtClientImpl,
-              atClientManager: mockAtClientManager,
-              monitor: mockMonitor) as NotificationServiceImpl;
+    test(
```

Comment:
Just in words can we document what tests were written to test the code?

Ex:

Test 1: Sync when local is ahead and check that the right commit id is sent to the server
Test 2: Sync when the cloud is ahead and verify that the right commit id is sent to the server
Test 3: Repeat a put and delete 100 times and verify the commit id

PS: Tests are just an example

Copied from comment I made on Murali's PR. Just an example.